base/lib/db.mustache

```diff
@@ -1,6 +1,6 @@
 const HOST = process.env.RETHINK_HOST;
 const PORT = process.env.RETHINK_PORT;
-const DB_NAME = process.env.RETHINK_NAME;
+const DB_NAME = `${process.env.RETHINK_NAME}${process.env.NODE_ENV === 'test' ? '_test' : ''}`;
```

Comment:
Maybe we want a function to do this? Then we can use it in all 4 places. I would have suggested lib/db, but that only exports a single function which is the database at the moment. `lib/db_helper` ?